lib/cdo/shared_constants.rb

```diff
@@ -772,4 +777,6 @@ module SharedConstants
     KAREN: "gen-ai-karen-creative-mistral-7b",
     PIRATE: "gen-ai-mistral-pirate-7b"
   }
+
+  US_STATES = STATE_ABBR_WITH_DC_HASH.merge(DC: 'Washington, D.C.').sort_by(&:last).to_h.freeze
```

Comment:
Changed the title for `DC` from **_"Washington DC"_** to **_"Washington, D.C."_**
https://github.com/code-dot-org/code-dot-org/blob/99520001a3dc5c48e17b2c63e2fa9b62b8c52d3d/lib/state_abbr.rb#L104